packages/math-input/package.json

```diff
@@ -44,7 +44,7 @@
         "@khanacademy/mathjax-renderer": "^2.0.0",
         "@khanacademy/wonder-blocks-clickable": "4.2.1",
         "@khanacademy/wonder-blocks-core": "6.4.0",
-        "@khanacademy/wonder-blocks-popover": "3.2.2",
+        "@khanacademy/wonder-blocks-popover": "3.2.9",
```

Comment:
**todo**: our WB dependencies in Perseus are a bit annoying to manage.  We need to keep the `devDependencies` in sync with the `peerDependencies` later in the same file.

The `devDependencies` are what our build/test/storybook tooling look at, but the `peerDependencies` help to notify consuming apps (webapp) what our base level of the dependency is.